client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -102,9 +111,58 @@ export default protectForm( React.createClass( {
 			placeholderText = this.translate( 'Loading' );
 		}
 
+		const {
+			site,
+			enableForm,
+			showUpgradeNudge,
+			jetpackModuleActive,
+			jetpackVersionSupportsModule,
+		} = this.props;
+
+		const {
+			domain = '',
+			slug = '',
+			jetpack = false,
+		} = site;
+
+		const isJetpackUnsupported = jetpack && ! jetpackVersionSupportsModule && isEnabled( 'jetpack/google-analytics' );
+
 		return (
 			<form id="site-settings" onSubmit={ this.handleSubmitForm } onChange={ this.props.markChanged }>
-				{ this.renderNudge() }
+
+				{ showUpgradeNudge &&
+					<UpgradeNudge
+						title={ this.translate( 'Add Google Analytics' ) }
+						message={ this.translate( 'Upgrade to the business plan and include your own analytics tracking ID.' ) }
```

Comment:
For some reason the business plan in Jetpack is called `Jetpack Professional`. How about this string:

`Upgrade to the Professional Plan to enable Google Analytics on your site.`